static/js/publisher/release/components/releasesTable/releaseCell.js

```diff
@@ -93,11 +95,16 @@ const ReleasesTableReleaseCell = props => {
   };
 
   function handleHistoryIconClick(arch, risk, track, branchName) {
+    triggerGAEvent("click-history", `${track}/${risk}/${branchName}/${arch}`);
```

Comment:
This would probably be better dispatched from the toggleHistoryPanel action - you'd have all the information you needed and also whether it's an open or close of the history.